app/views/idv/doc_auth/ssn.html.erb

```diff
@@ -1,5 +1,12 @@
 <% title t('doc_auth.titles.doc_auth') %>
 
+<%= render 'shared/alert', {
+  type: 'success',
+  class: 'margin-bottom-4 margin-top-2 tablet:margin-top-0'
```

Comment:
The additional margins here were because the natural margin above the notice is quite small (smaller than designed) in mobile displays. However, it has me wondering if, rather than adding margins ad hoc like this, if we'd want to consider _increasing_ the default padding of the body container in the mobile stylesheet.

Not in scope for this issue, but an idea which occurred to me nonetheless.

cc @anniehirshman-gsa

Reply:
@aduth thanks for flagging this!  Yes, let's keep this in the back of our minds address in either the spacing or typography updates.